src/main/java/com/google/ehub/utility/NextIncreasingLexicographicStringUtility.java

```diff
@@ -0,0 +1,37 @@
+package com.google.ehub.utility;
+
+/**
+ * Utility class that holds method to find the next lexicographic string.
+ */
+public final class NextIncreasingLexicographicStringUtility {
```

Comment:
Let's start a github issue and keep track of all the classes we need to write tests for

Reply:
I created a new issue about the missing tests for most of the Servlet code.